src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Repository/DevAdventCalendarCompetition.Repository.csproj

```diff
@@ -18,15 +22,10 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.2.6" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="2.2.6">
```

Comment:
why have you removed it?

Reply:
It brings warning during building application ("NETSDK1059 : The tool 'Microsoft.EntityFrameworkCore.Tools.DotNet' is now included in the .NET Core SDK.)

Reply:
aha ok ;)